config/database.yml

```diff
@@ -1,36 +1,70 @@
-default: &default
+
+shared_db_settings: &shared_db_settings
   adapter: postgresql
-  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  encoding: unicode
+
+default_primary: &default_primary
```

Comment:
just wanted to confirm this has been testing on all envs in here to ensure connections aren't lost?

Reply:
I've tested on the review environment and migration environment (which has the same config as sandbox/staging/prod). I think the only way it could fall over is if there is an issue with the production credentials, but we can't really test that short of deploying to prod 😬